tensorboard/uploader/uploader.py

```diff
@@ -720,6 +727,14 @@ def _get_or_create_blob_sequence(self):
     def _send_blob(self, blob_sequence_id, seq_index, blob):
         # TODO(soergel): retry and resume logic
 
+        if len(blob) > self._max_blob_size:
+            logger.warn(
```

Comment:
The `warn` method is deprecated; please use `warning`. (This should have
printed a `DeprecationWarning` in tests?)

Reply:
Huh, no `DeprecationWarning` that I noticed on a quick skim.  Done, anyway.

Reply:
Interesting. I see it on Python 3.7.6:

> uploader.py:741: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead

Perhaps we have different environments somehow. Thanks, anyway.